samples/monster_generated.h

```diff
@@ -199,11 +202,11 @@ struct Monster FLATBUFFERS_FINAL_CLASS : private flatbuffers::Table {
     VT_EQUIPPED_TYPE = 20,
     VT_EQUIPPED = 22
   };
-  const Vec3 *pos() const {
+  const Vec3 * _Nullable pos() const {
```

Comment:
This particular file needs to be compile with other compilers. Either --clang-nullable should be by default off for these test files, or we should `#define _Nullable` for non-clang compilers.

Reply:
yes, currently --clang-nullable is off by default, I can revert the generate script.